lib/render/editor/editor.html

```diff
@@ -284,6 +284,7 @@
       _onExecutionStop: function(data) {
         this._cleanState();
         mumuki.showKidsResult(data);
+        mumuki.updateProgressBarAndShowModal(data);
```

Comment:
Why should the runner be so tightly coupled to laboratory? Why can't `showKidsResult` already update progress bar if needed?

By handling the UI flow here, we will have to change it in both laboratory and any runner that supports kids layout. 

And it even gets worse: why should the custom editor of the runner know about kids mode? The custom editor and the kids mode are separate, orthogonal concerns.

Reply:
We can include the call to updateProgress... inside showKidsResults.
I wanted to make them independent in case we just need to use showKidsResults from labo js.

I agree that they are more coupled than i would like. We should implement a way for each layout to expose a resultArea, and let the editor set the result there without knowing which layout is being used.

Nevertheless, i think that they are coupled regardless of this commit and we should refactor it a bit in the future.
What do you think about doing what you said (updateProg inside showKids) and postpone the refactor until the sea is more calmed? :stuck_out_tongue: